inc/html.class.php

```diff
@@ -3048,18 +3025,19 @@ static function getGenericDateTimeSearchItems($options) {
    }
 
 
-    /**
+   /**
     * Compute date / datetime value resulting of showGenericDateTimeSearch
     *
     * @since 0.83
     *
-    * @param $val          date / datetime   value passed
-    * @param $force_day    boolean           force computation in days (false by default)
-    * @param $specifictime timestamp         set specific timestamp (default '')
+    * @param string  $val            date / datetime value passed
+    * @param boolean $force_day      force computation in days (false by default)
+    * @param string  $specifictime   set specific timestamp (default '')
     *
-    * @return computed date / datetime value
-    * @see showGenericDateTimeSearch
-   **/
+    * @see   showGenericDateTimeSearch
+    *
+    * @return false|string
```

Comment:
Should only return string. Returning false is a bug and should be handled by function itself.

Reply:
you are returning the result of the date and it returns false or string, so this method does the same